src/wp-includes/js/admin-bar.js

```diff
@@ -10,12 +10,30 @@ if ( typeof(jQuery) != 'undefined' ) {
 	jQuery(document).ready(function($){
 		var adminbar = $('#wpadminbar'), refresh, touchOpen, touchClose, disableHoverIntent = false;
 
+		/**
+		 * @summary Forces the browser to refresh the tabbing index.
+		 *
+		 * @since 3.5.0
+		 *
+		 * @param  {integer} i The index of the element.
+		 * @param {HTMLelement} el  The HTML element to remove the tab index from.
+		 * @returns {void}
```

Comment:
Please add a newline between params and returns
https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions